opml-importer.php

```diff
@@ -71,15 +73,20 @@ function dispatch() {
 
 </div>
 
+<?php
+$categories = get_terms('link_category', array('get' => 'all'));
+
+if ( count($categories) ) {
```

Comment:
If you're going to add defensive coding...
* Should a check for `WP_Error` being returned be added ?
* `get_terms()` does not always return an array, so `count()` on its own is not sufficient. Maybe check with `is_array() && ! empty()` ?

Ref: https://developer.wordpress.org/reference/functions/get_terms/